### PR TITLE
Add 'sw_64' cpu support

### PR DESCRIPTION
**Pull request by KunWuChan**
add sw_64 cpu support for 'Shenwei/Sunway CPU architecture'

### CODE REVIEW

File: virtManager/createvm.py [hunk 1]
```diff
@@ -858,6 +859,9 @@ def _populate_machine(self):
 
         defmachine = None
         prios = []
+        if self._capsinfo.arch in ["sw_64"]:
+            defmachine = "core3"
```
Comment (ptoscano) [inline virtManager/createvm.py:863]: i think this should be done in `virtinst.Guest.get_recommended_machine()`, in the internal `_qemu_machine()` function of it: see the existing cases

Comment (KunWuChan) [reply virtManager/createvm.py]: Thanks for the kind reminder, I've fixed it

File: virtManager/createvm.py [hunk 2]
```diff
@@ -858,6 +859,7 @@ def _populate_machine(self):
 
         defmachine = None
         prios = []
+
```
Comment (ptoscano) [inline virtManager/createvm.py:862]: extra/unneeded change

**Review by ptoscano — COMMENTED** [review]
other than the things I already pointed out, I'm not familiar enough with these CPU-related bits to know whether these "sw_64" changes are OK; I'll leave this to @crobinso

**Comment by crobinso** [comment]
This seems to be about non-upstream qemu support. Before putting code into virt-manager we need upstream qemu and libvirt to support your feature. After sw_64 support is upstream, please re-file this PR. Thanks!

**Pull request closed without merging**